kernel/kernbench.py

```diff
@@ -55,7 +55,7 @@ def time_build(self, threads=None, timefile=None, make_opts=None):
         else:
             build_string = "/usr/bin/time -o %s make -j %s vmlinux" % (
                 timefile, threads)
-        process.system(build_string, allow_output_check='None',
+        process.system(build_string, allow_output_check=None,
```

Comment:
I think we can remove this param as default value is None.

Reply:
yes make sense will do